system/jlib/jstats.h

```diff
@@ -510,12 +510,18 @@ class CNestedRuntimeStatisticMap;
 class jlib_decl CRuntimeStatisticCollection
 {
 public:
-    CRuntimeStatisticCollection(const StatisticsMapping & _mapping) : mapping(_mapping)
-    {
+    CRuntimeStatisticCollection(const StatisticsMapping & _mapping, bool _filtering = false) : mapping(_mapping)
```

Comment:
would probably name the parameter ignoreUnknownStats or similar, and not have the conditional compiles.